tree.js

```diff
@@ -0,0 +1,191 @@
+/* eslint-disable import/prefer-default-export */
+
+class Node {

```

Comment:
I think this file is not pretty formatted, can you check? have you integrated prettier?

Reply:
I made the neccassary changes.